sdk/containerregistry/pom.xml

```diff
@@ -0,0 +1,57 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-containerregistry-service</artifactId>
+  <packaging>pom</packaging>
+  <version>1.0.0</version><!-- Need not change for every release-->
+  <profiles>
+    <!-- DO NOT CHANGE THE ORDER IN WHICH THESE PROFILES ARE DEFINED -->
+    <profile>
+      <id>coverage</id>
+      <modules>
+        <module>azure-containers-containerregistry</module>
+      </modules>
+      <dependencies>
+        <dependency>
+          <groupId>com.azure</groupId>
```

Comment:
Why do we need to add a dependency to itself?

Reply:
Found it, we use this field to do version scanning.

Reply:
This is used for generating code coverage reports. Since this is an [aggregator pom](https://maven.apache.org/pom.html#aggregation-or-multi-module), we need to explicitly add dependencies so that the aggregate code coverage is created for all the modules in this directory.